### ISSUE TITLE
anti kick [Feature]

### ISSUE THREAD

**Issue by tipten** [IcyRelic/LethalMenu#13]
you should add anti kick to troll people and make a mod so that u can respawn

**Comment by FancyDucc** [comment]
I dont think this is actually possible, since being kicked is all server-side, the only thing that would be client-side is the host pressing the X next to your name and sending a signal to the server

**Comment by IcyRelic** [comment]
The version that exists currently from another menu only works if you are the last user to join a lobby and it makes it so your name is random, Unknown or Nameless. Its not a very user friendly experience. I have tried many different things to implement a similar feature without any luck. 

I wont be implementing the current methods of antikick due to its quirks but will keep this feature in the backlog and if a better way is found it will defiantly make its way to LethalMenu
